example/lib/widgets/modal_bottom_sheet.dart

```diff
@@ -237,8 +237,8 @@ Future<T?> showModalBottomSheet<T>({
       ));
 }
 
-PersistentBottomSheetController<T> showMyBottomSheet<T>({
+PersistentBottomSheetController showMyBottomSheet<T>({
```

Comment:
is this change related to the SDK upgrades?

Reply:
Yes,

Reply:
Yes, With latest flutter version the <T> datatype is not support as there are some changes in Dart code.